### PR TITLE
add output and fix output that is cut off the slide

### PR DESCRIPTION
**Pull request by patkearns10**
Couple quick things that were broken or missing

### CODE REVIEW

File: _lessons/intro-to-jinja.html [hunk 1]
```diff
@@ -195,6 +206,24 @@ <h1 style="font-size: 300%;" class="remark-inline-code">
 ```
 foo
 ```
+---
```
Comment (patkearns10) [inline _lessons/intro-to-jinja.html:209]: this command / output was cut off the bottom of the slide, so I added a new slide for the second command / output

File: _lessons/intro-to-jinja.html [hunk 2]
```diff
@@ -99,6 +99,17 @@ <h1 style="font-size: 300%;" class="remark-inline-code">
 {% set me = 3 %}
 {% set you = 'foo' %}
 ```
+--
```
Comment (patkearns10) [inline _lessons/intro-to-jinja.html:102]: there was no command / output for these variables, so adding that in.

**Comment by alexiswo** [comment]
Hey @patkearns10 thanks for making these changes! We just finished migrating this repo into the www repo. Would you mind closing this PR and re-creating it on https://github.com/dbt-labs/www.getdbt.com? Check out the /_lessons folder. It should easier to test locally if you want to try that too!

**Comment by patkearns10** [comment]
Should we deprecate this repo or lock it?

**Pull request closed without merging**